puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/releaseTarballs.groovy

```diff
@@ -0,0 +1,61 @@
+pipeline {
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        parallel {
+            stage('Foreman') {
+                steps {
+                    verify_tag('foreman')
+                    build_tarball('foreman')
+                }
+            }
+            stage('Smart Proxy') {
+                steps {
+                    verify_tag('smart-proxy')
+                    build_tarball('smart-proxy')
+                }
+            }
+            stage('Foreman SELinux') {
+                steps {
+                    verify_tag('foreman-selinux')
+                    build_tarball('foreman-selinux')
+                }
+            }
+            stage('Foreman Installer') {
+                steps {
+                    verify_tag('foreman-installer')
+                    build_tarball('foreman-installer')
+                }
+            }
+        }
+    }
+
+    post {
+        success {
+            build job: 'release_packages', parameters: [
+                [$class: 'StringParameterValue', name: 'version', value: env.getProperty('version')],
+                [$class: 'StringParameterValue', name: 'major_version', value: env.getProperty('major_version')]
+            ]
+        }
+    }
+}
+
+void verify_tag(project) {
+    ext_job('release_verify_tag', project)
+}
+
+void build_tarball(project) {
+    ext_job('release_build_tarball', project)
+}
+
+void ext_job(project, job) {
```

Comment:
What is ext short for?

Reply:
"external job" as in a job outside of this pipeline.